src/dropzone/DropzoneDirective.ts

```diff
@@ -57,10 +38,8 @@ export class DropzoneDirective implements ng.IDirective {
     let eventHandlers = {
       'sending': (file, xhr, formData) => {
         scope.$apply(() => {
-          // s3 file upload
-          if (scope.$ctrl.backend.hasOwnProperty('s3')) {
-            uploadToS3(scope.$ctrl, file, formData);
-          }
+          // set form data prior to submitting the dz form
+          scope.$ctrl.backend.setFormData(scope.$ctrl.folder, file, formData);
```

Comment:
This commit also looks like it should be "fixup"ped into the previous commit in this branch where Alex pointed out the original problem.